src/server/pegasus_server_impl.h

```diff
@@ -297,8 +300,38 @@ class pegasus_server_impl : public ::dsn::apps::rrdb_service
         return false;
     }
 
+    ::dsn::error_code check_meta_cf(const std::string &path, bool *need_create_meta_cf);
+
+    enum class MetaStoreType
```

Comment:
Can we add a new class for meta column store? Current pegasus_server_impl is too bloated. Too many responsibilities in one single class.

```cpp
class meta_column_store 
{
public:
  enum class MetaStoreType
  {
      kManifestOnly = 0,
      kMetaCFOnly,
      kBothManifestAndMetaCF,
  }

  meta_column_store()
  {
  }

  uint64_t get_last_flushed_decree(MetaStoreType type) const;
  uint32_t get_data_version(MetaStoreType type) const;
  uint64_t get_last_manual_compact_finish_time(MetaStoreType type) const;

  dsn::error_code get_value_from_meta_cf(rocksdb::DB *db,
                                         rocksdb::ColumnFamilyHandle *cfh,
                                         bool read_flushed_data,
                                         const std::string &key,
                                         uint64_t *value) const;
  dsn::error_code
    set_value_to_meta_cf(rocksdb::DB *db, const std::string &key, uint64_t value) const;

private:
    rocksdb::ColumnFamilyHandle *_cf;
    rocksdb::ColumnFamilyOptions _cf_opts;
};
```

Reply:
MetaStoreType => meta_store_type